browser/ui/brave_shields_data_controller.h

```diff
@@ -40,6 +44,9 @@ class BraveShieldsDataController
    public:
     virtual void OnResourcesChanged() = 0;
     virtual void OnFaviconUpdated() {}
+    // Fired only when current current tab's shields settings are changed
+    // by different tab.
+    virtual void OnContentSettingsChangedByOthers(ContentSettingsType type) {}
```

Comment:
I'm not complete sure how about this long name. i'd probably make it sound within shields context as other observer methods are following those. ex - `OnResourcesChanged()`  - perhaps this should be `OnSiteSettingsChanged()` or `OnSettingsChanged()` 

i'd say to include "Shields" but that would be redundant since we're exporting this from the ShieldsDataController class